lib/test_case.dart

```diff
@@ -129,6 +129,23 @@ class TestCase {
     return TestCase.fromJson(response);
   }
 
+  static Future<SoftDeleteTestCase> delete(
```

Comment:
Why wouldn't this be an instance method? That way the code reads `testCase.delete(soft: true)`. And you'd have the `caseId` already accessible on the instance.

Reply:
Good point, thanks